index.html

```diff
@@ -224,36 +224,67 @@
+          resulting media stream has a single
+          video <code>MediaStreamTrack</code> that matches the dimensions of the
+          canvas element.
         </p>
         <p>
-          This method throws a <a
-          href="http://www.w3.org/html/wg/drafts/html/master/infrastructure.html#securityerror">SecurityError</a>
-          exception if the canvas is not <dfn><a
-          href="http://www.w3.org/html/wg/drafts/html/master/semantics.html#concept-canvas-origin-clean">origin-clean</a></dfn>.
+          This method throws
+          a <a href="http://www.w3.org/html/wg/drafts/html/master/infrastructure.html#securityerror">SecurityError</a>
+          exception if the canvas is
+          not <dfn><a href="http://www.w3.org/html/wg/drafts/html/master/semantics.html#concept-canvas-origin-clean">origin-clean</a></dfn>.
           The captured stream immediately ceases to capture content from the
-          canvas if the <a>origin-clean</a> flag of the canvas becomes false at
-          any time.
+          canvas if the <a>origin-clean</a> flag of the canvas becomes false.
         </p>
         <p>
-          A <a>user agent</a> SHOULD <a
-          href="http://www.w3.org/TR/html5/webappapis.html#await-a-stable-state">await
-          a stable state</a> in the script execution of the current page or
-          worker that has control of the canvas before capturing a frame.
+          Each track that captures a canvas has
+          a <code><dfn>frameCaptureRequested</dfn></code> property that is set to
+          true when a new frame is requested from the canvas.
+        </p>
+        <p>
+          The value of the <code><a>frameCaptureRequested</a></code> property on
+          all new tracks is set to <code>true</code> when the track is created.
+          On creation of the captured track with a specific,
+          non-zero <code>frameRate</code>, the <a>user agent</a> starts a
+          periodic timer at an interval of <code>1/frameRate</code> seconds.  At
+          each activation of the timer,
+          the <code><a>frameCaptureRequested</a></code> property is set
+          to <code>true</code>.
         </p>
         <p>
           In order to support manual control of frame capture, browsers MUST
-          support a value of 0 for <code>frameRate</code>.  The captured stream
-          always captures at least one frame, even if <code>frameRate</code> is
-          zero.
+          support a value of 0 for <code>frameRate</code>.
         </p>
         <p>
-          If the <code>frameRate</code> value is omitted, the <a>user agent</a>
-          SHOULD capture new frames each time that the content of the canvas
-          changes.
+          A new frame is requested from the canvas
+          when <code><a>frameCaptureRequested</a></code> is true and the canvas is
+          drawn.  Each time that the captured canvas is drawn, the following
```

Comment:
Say rendered instead of drawn here and at most other places.

I think the right terminology is that JS draws multiple times to build a frame, and that is then rendered all at once.